ssn/rdf/examples/unused/vistavital-a-sdo.ttl

```diff
@@ -95,7 +95,7 @@ vistavital-a:Observation_1_ex
   rdfs:seeAlso <https://github.com/vistadataproject/documents/blob/master/Background/presentations/pdf/2014-Linked_Vitals-Linked_Data_Linked_FHIR.pdf> ;
   prov:atLocation <https://example.org/HOSPITAL-LOCATION/4-SOUTH-MED> ;
   sosa:hasFeatureOfInterest <https://example.org/patient/jones-christopher> ;
-  sosa:hasResult "150/10" ;
+  sosa:hasSimpleResult "150/10" ;
```

Comment:
These simple results would benefit from a Datatype.

Reply:
This discussion belongs in https://github.com/w3c/sdw-sosa-ssn/pull/320